helm-chart/values.yaml

```diff
@@ -9,7 +23,7 @@ images:
 
 serviceAccounts:
   # Specifies whether service accounts should be created.
-  create: false
+  create: true
```

Comment:
**Question:** Was this intentional?

Reply:
Yes, service accounts are a good best practice and we decided to turn them on by default in kustomize-- might as well reflect that in Helm as well.

Reply:
Sounds good! We should eventually:
1. remove the ServiceAccounts Kustomize component + add it to the default manifests
2. remove the option to remove ServiceAccounts in Helm (i.e., `create: false` seems unnecessary).

Reply:
Created https://github.com/GoogleCloudPlatform/microservices-demo/issues/2508